ash/src/lib.rs

```diff
@@ -45,6 +50,11 @@
 //! * **debug** (default): Whether Vulkan structs should implement `Debug`.
 //! * **loaded** (default): Support searching for the Vulkan loader manually at runtime.
 //! * **linked**: Link the Vulkan loader at compile time.
+//! * **std** (default): Whether ash depends on the standard library (otherwise alloc is required)
+
+#[cfg_attr(feature = "std", allow(unused_extern_crates))]
```

Comment:
Is this `allow` needed? Looks like we have unconditional references to `alloc::` scattered around, which is the norm for no_std crates.